test/check_an_util.c

```diff
@@ -0,0 +1,125 @@
+#include <assert.h>
+#include <check.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "an_util.h"
+
+START_TEST(test_an_next_power_of_2)
+{
+	ck_assert(an_next_power_of_2(1) == 1);
```

Comment:
I'd rather have a property:

1. the value is a power of two;
2. the value is >= the input;
3. the value / 2 is < the input.

and we could easily enumerate all ints up to 65536.